### DISCUSSION TITLE
Discrepancy in Training Times

### DISCUSSION THREAD

**Discussion by ZeeRizvee** [ultralytics/ultralytics#13396] [Q&A]
Hi there! I have used both Ultralytics and MMYOLO to run training of different models to compare performance. Running the same model (YOLOv5s) through Ultralytics takes considerably less time than training with MMYOLO. This is quite confusing when the same machine is used to train similar models. Can somebody help me understand this?

**Comment by glenn-jocher** [comment]
@ZeeRizvee hello!

Great observation! The difference in training times between Ultralytics YOLOv5 and MMYOLO can be attributed to several factors, including the implementation details and optimizations specific to each framework. Ultralytics YOLO models are highly optimized for both speed and efficiency, which might explain the faster training times you're experiencing.

If you're looking to delve deeper into the specifics, you might want to compare the configurations and underlying code for data loading, augmentation, and model architecture adjustments between the two frameworks. This could provide more insights into the performance differences.

If you have any more questions or need further assistance, feel free to ask!

Happy training! 🚀
